src/mobo_qm9.py

```diff
@@ -98,11 +102,13 @@ def get_surrogate_model(self, X, y, kernel_type='RBF'):
         else:
             raise ValueError("Unsupported kernel type. Supported types are 'RBF', 'Matern', and 'Tanimoto'.")
 
-        model = SingleTaskGP(train_X, train_Y, likelihood=likelihood, kernel=kernel)
+        models = [FixedNoiseGP(train_X_scaled, train_Y, noise=torch.zeros_like(train_Y), likelihood=likelihood, kernel=kernel)]
+
+        model = ModelListGP(*models)
         mll = gpytorch.mlls.ExactMarginalLogLikelihood(likelihood, model)
         fit_gpytorch_model(mll)
-    
-        return model
+
+        return model, input_transform
```

Comment:
Please check whether this is correct, I am bit doubtful regarding the function returning